db/seeds.rb

```diff
@@ -50,3 +50,20 @@
     event_id += 1
   end
 end
+
+event_id = 0
+ConferenceRoom.all.each do |conference_room|
+  params = {
+      start_time: Time.now.beginning_of_day + 6.hours,
+      end_time: Time.now.beginning_of_day + 7.hours,
+      name: Faker::Company.name,
+      description: Faker::Company.catch_phrase,
+      conference_room: conference_room,
+      user: Faker::Name.name
+  }
+  if(event = Event.all[event_id])
```

Comment:
This will create the following SQL query:

`SELECT * from events`

I think you prefer to use the following:

`SELECT * from events WHERE id = event-id`

Use `Event.find(event_id)` instead